frontend/client/src/pages/Order/AddressInfo/AddressSelectModal/AddressSelectModal.tsx

```diff
@@ -0,0 +1,128 @@
+import Button from '@src/components/Button/Button';
+import { Address } from '@src/types/Address';
+import { FiX } from 'react-icons/fi';
+import React from 'react';
+import styled from 'styled-components';
+import AddressCard from '../AddressCard/AddressCard';
+import { useRef } from 'react';
+import Portal from '@src/portal/portal';
+
+interface Props {
+  addressList: Address[];
+  onSelect: (addressId: number) => void;
+  onCancel: () => void;
+}
+
+// TODO: 노데이터
+const AddressSelectModal: React.FC<Props> = ({ addressList, onSelect, onCancel }) => {
+  const dim = useRef(null);
```

Comment:
ref 변수이면 표현해주는것이 좋을 것 같아요! 
```suggestion
  const dimRef = useRef<HTMLDivElement>(null);
```

Reply:
동의합니다!! ㅎㅎㅎ 와 근데 PR code suggest 기능이 있었군요!! 되게 신기하다